docs/unified/novacustom/recovery.md

```diff
@@ -81,102 +141,288 @@ make BOARD=arduino/mega2560
 make BOARD=arduino/mega2560 flash
```

Comment:
this doesn't work for me:

```
flewinski in ~/workstation/ec_reflash/ec on master λ make BOARD=arduino/mega2560
make BOARD=arduino/mega2560 flash
  CC        arduino/mega2560/2024-07-17_4ae73b9d2cc2/arch/avr/gpio.o
  CC        arduino/mega2560/2024-07-17_4ae73b9d2cc2/arch/avr/i2c.o
  CC        arduino/mega2560/2024-07-17_4ae73b9d2cc2/arch/avr/i2c_slave.o
src/arch/avr/i2c_slave.c: In function '__vector_39':
src/arch/avr/i2c_slave.c:73:13: error: array subscript 0 is outside array bounds of 'volatile uint8_t[0]' {aka 'volatile unsigned char[]'} [-Werror=array-bounds=]
   73 |             i2c_slave_recv_cb(TWDR);
      |             ^~~~~~~~~~~~~~~~~~~~~~~
cc1: note: source object is likely at address zero
cc1: all warnings being treated as errors
make: *** [src/arch/avr/toolchain.mk:42: build/arduino/mega2560/2024-07-17_4ae73b9d2cc2/arch/avr/i2c_slave.o] Error 1
  CC        arduino/mega2560/2024-07-17_4ae73b9d2cc2/arch/avr/i2c_slave.o
src/arch/avr/i2c_slave.c: In function '__vector_39':
src/arch/avr/i2c_slave.c:73:13: error: array subscript 0 is outside array bounds of 'volatile uint8_t[0]' {aka 'volatile unsigned char[]'} [-Werror=array-bounds=]
   73 |             i2c_slave_recv_cb(TWDR);
      |             ^~~~~~~~~~~~~~~~~~~~~~~
cc1: note: source object is likely at address zero
cc1: all warnings being treated as errors
make: *** [src/arch/avr/toolchain.mk:42: build/arduino/mega2560/2024-07-17_4ae73b9d2cc2/arch/avr/i2c_slave.o] Error 1
```

ran from ec-sdk: 

```
root@3b0108acf9bb:/home/flewinski/workstation/ec_reflash/ec# make BOARD=arduino/mega2560 flash
make: sudo: Command not found
make: *** [src/board/arduino/mega2560/board.mk:19: flash] Error 127
root@3b0108acf9bb:/home/flewinski/workstation/ec_reflash/ec#
```

It **did** work when I ran the first command from `ec-sdk` container, the second from outside, having removed the `sudo` from `src/board/arduino/mega2560/board.mk:19`